src/lib/util/calculateShopBuyCost.ts

```diff
@@ -0,0 +1,25 @@
+export function calculateShopBuyCost(
+	gpCost: number,
+	quantity: number,
+	shopQuantity?: number,
+	changePer?: number
+): { total: number; average: number } {
```

Comment:
This should be changed to an object for the parameters, so that its less prone to accidentally passing them in the wrong order.

```ts
export function calculateShopBuyCost({
	gpCost,
	quantity,
	shopQuantity,
	changePer
}: { gpCost: number; quantity: number; shopQuantity?: number; changePer?: number }): {
	total: number;
	average: number;
} {```